Add/update Harvester landing page (#940)
rancher/rancher-docs: docs/integrations-in-rancher/harvester/harvester.md

---
title: Virtualization on Kubernetes with Harvester
---

## Harvester

> Introduced in Rancher v2.6.1, Harvester is an open-source hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require users to know Kubernetes concepts, making it a more user-friendly application.

Reviewer: ```suggestion
Introduced in Rancher v2.6.1, Harvester is an open-source, hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require knowledge of Kubernetes concepts, making it more user-friendly.
```

## Harvester with Rancher

> With Rancher Prime and Harvester, IT operators now have access to an enterprise-ready, simple-to-use infrastructure platform that cohesively manages their virtual machines and Kubernetes clusters alongside one another. For more information on the support offering, see the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/). With the Rancher Virtualization Management feature, users can import and manage multiple Harvester clusters. Leveraging the Rancher's authentication feature and RBAC control for multi-tenancy support.

Reviewer: ```suggestion
With Rancher Prime and Harvester, your IT operators have an enterprise-ready, simple-to-use infrastructure platform that cohesively manages their virtual machines and Kubernetes clusters alongside one another. 

Rancher's Virtualization Management empowers your team to import and manage multiple Harvester clusters, and leverage Rancher's authentication feature and RBAC control for multi-tenancy support.

See the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/) for more details about hardware requirements, supported software, and the Harvester support lifecycle. 
```

Who is the audience for this page? I'm taking it as, people who have the power to get their team to use Rancher and Harvester.

Author: @CamrynCarter Do we have a specific target audience for the various landing pages?
